src/pages/user_profile/RequestForm.js

```diff
@@ -44,19 +43,6 @@ export default function SupportRequestsPage(props) {
       extraRequest: undefined
     };
   }
```

Comment:
The previous if condition here (`if (props.isRequestRolePage)`) is always true so we don't need that any longer. It was only used to distinguish between the LC request vs Role request.